example/basic_example.py

```diff
@@ -96,73 +98,69 @@ def parse_arguments():
             "OneAgent endpoint.")
 
     # set up OpenTelemetry for export:
+    # This call sets up the MeterProvider, with a PeriodicExportingMetricReader that exports every 5000 ms
+    # and the Dynatrace exporter exporting to args.endpoint with args.token
     logger.debug("setting up global OpenTelemetry configuration.")
-    metrics.set_meter_provider(MeterProvider())
-    meter = metrics.get_meter(splitext(basename(__file__))[0])
+    _metrics.set_meter_provider(MeterProvider(
+        metric_readers=[PeriodicExportingMetricReader(
+            export_interval_millis=5000,
+            exporter=DynatraceMetricsExporter(args.endpoint, args.token,
+                                              prefix="otel.python",
+                                              export_dynatrace_metadata=args.metadata_enrichment))]))
+
+    meter = _metrics.get_meter(splitext(basename(__file__))[0])
 
-    logger.info("setting up Dynatrace metrics exporting interface.")
     exporter = DynatraceMetricsExporter(args.endpoint, args.token,
```

Comment:
I guess we can remove this, right? Since it's instantiated above when creating the `MeterProvider`